es_find_block.py

```diff
@@ -23,15 +23,15 @@ class Limitable():
     def query_args(self, signal=None):
         existing_args = super().query_args(signal)
 
-        size = int(self.size(signal))
+        size = self.size(signal)
         if size:
             # don't send size parameter if 0 is passed
```

Comment:
This comment seems incorrect after this change. '0' will fall into this if block and be added.